README.md

```diff
@@ -45,6 +45,8 @@ Create a `.env` file under the root project directory and populate it with the f
 export PORT=6060
 export CLIENT_ORIGIN_URL=http://localhost:4040
 export SECRET_KEY_BASE=your_secret_key
+export AUTH0_DOMAIN=auth0_domain
+export AUTH0_AUDIENCE=auth0_audience
```

Comment:
@manusajith I think I left a similar question on the other branch, but why `export` is required here?